charts/datadog/templates/NOTES.txt

```diff
@@ -580,3 +580,11 @@ You are using the datadog.securityAgent.compliance.xccdf.enabled parameter which
 This version still supports both but the support of the old name will be dropped in the next major version of our Helm chart.
 More information about this change: https://github.com/DataDog/helm-charts/pull/1161
 {{- end }}
+
+
+{{- if and .Values.datadog.otelCollector.enabled .Values.providers.gke.autopilot }}
```

Comment:
Since you have implement a helper function (https://github.com/DataDog/helm-charts/pull/1429/files#diff-483dad0ace7ed469efb9d93ba2f3d3c2045bbc1cd317f49e7e0a122bdf9d6116R938) to know if the otel-agent need to be enable or not.
IMO this `if` condition should use it

Reply:
done